test/matching.js

```diff
@@ -1,8 +1,8 @@
 var assert = require("assert");
 const { matchPeople } = require("../src/matching");
 
-function person(id, added = true) {
-  return { id: id, added: added };
+function person(id, added = true, location = "", team = "") {
+  return { id: id, added: added, location: location, team: team };
 }
```

Comment:
In JS if variable is equal to the name of the property you can do shorthand assignment like this, and also maybe change to arrow func:

```suggestion
const person = (id, added = true, location = "", team = "") => (
  { id, added, location, team }
)
```

Reply:
thx, I always forget about this feature ;)
Although I'm not a fan of arrow functions when using them as definitions